android/build.gradle

```diff
@@ -60,6 +60,7 @@ if (androidSdkInstalled) {
                             "-DBORINGSSL_HOME=$boringsslHome",
                             "-DCMAKE_CXX_STANDARD=17"
                     cFlags '-fvisibility=hidden',
+                            '-Wl,-z,max-page-size=16384',
```

Comment:
I'd suggest putting linker flags at the end of the list.

Reply:
Actually turns out this isn't the way to do it with external cmake,  only actual cflags can do in here.

You want to add `'-DCMAKE_SHARED_LINKER_FLAGS=-z max-page-size=16384'` to the cmake arguments.

Reply:
Done, and thank you thank you, I had to do something slightly differently, but I think this should work now.

Reply:
Hmm.  CMAKE_SHARED_LINKER_FLAGS needed to be exactly as I quoted (it's already a linker flag so -Wl is redundant... Odd that it didn't error out in CI or local builds as I'm sure it did for me on the 2.5.x branch. Anyways hopefully with that editted this PR should be doing the correct thing.

Reply:
Ah, that builds on MacOS and Windows but not Ubuntu... yay for consistency.

Reply:
Ah, you added it to cflags, not the Cmake arguments...